crates/trycp_server/src/main.rs

```diff
@@ -80,11 +82,22 @@ struct TrycpServer {
     port_range: PortRange,
 }
 
+fn make_conductor_dir() -> PathBuf {
+    let conductor_path = PathBuf::new().join(TRYCP_DIRNAME).join(CONDUCTORS_DIRNAME);
+    std::fs::create_dir_all(conductor_path.clone()).expect("can create conductors dir");
+    let dir = Builder::new().tempdir_in(conductor_path).expect("should create conductor tmp dir").into_path();
```

Comment:
This is always random so shouldn't ever fail due to uniqueness issues, unlike the previous step.